examples/chip-tool/commands/Commands.h

```diff
@@ -0,0 +1,48 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#ifndef __CHIPTOOL_COMMANDS_H__
+#define __CHIPTOOL_COMMANDS_H__
+
+#include "Echo.h"
+#include "EchoBle.h"
+#include "Off.h"
+#include "On.h"
+#include "ReadOnOff.h"
+#include "Toggle.h"
+
+Command * Commands[] = {
+    new Echo(), new EchoBle(), new On(), new Off(), new Toggle(), new ReadOnOff(),
+};
+
+Command * GetCommandForArgs(int argc, char * argv[])
+{
+    size_t commandsCount = sizeof(Commands) / sizeof(Commands[0]);
+    for (int i = 0; i < commandsCount; i++)
```

Comment:
consider using type `size_t` for `i` to match `commandsCount`

Reply:
Fixed.

Reply:
We could probably use a ranged for here too...